packages/adyen-integration/src/adyenv3/adyenv3-payment-strategy.ts

```diff
@@ -86,6 +86,12 @@ export default class Adyenv3PaymentStrategy implements PaymentStrategy {
                 klarna_paynow: {
                     useKlarnaWidget: true,
                 },
+                card: {
+                    installmentOptions: {
+                        showInstallmentAmounts: true,
+                        ...installmentOptions,
```

Comment:
It might be better to check if `installmentOptions` is an object first, because it might throw an error if the value will be null or undefined (or whatever value it might be except object)